variable/technology/power-plant.yaml

```diff
@@ -0,0 +1,799 @@
+                plant category
+   unit: TWh
+
+Operating Reserve|Down|Electricity|Nuclear:
+   description: Operating reserve down of a power generation unit who belongs to the specified power
+                plant category
+   unit: TWh
+
+Operating Reserve|Down|Electricity|Oil:
+   description: Operating reserve down of a power generation unit who belongs to the specified power
+                plant category
+   unit: TWh
+
+Operating Reserve|Up|Electricity|Coal:
+   description: Operating reserve up of a power generation unit who belongs to the specified power
+                plant category
+   unit: TWh
+
+Operating Reserve|Up|Electricity|Gas:
+   description: Operating reserve up of a power generation unit who belongs to the specified power
+                plant category
+   unit: TWh
+
+Operating Reserve|Up|Electricity|Nuclear:
+   description: Operating reserve up of a power generation unit who belongs to the specified power
+                plant category
+   unit: TWh
+
+Operating Reserve|Up|Electricity|Oil:
+   description: Operating reserve up of a power generation unit who belongs to the specified power
+                plant category
+   unit: TWh
+
+Maximum Ramping|Upwards|Electricity|Coal:
+   description: Upward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Maximum Ramping|Upwards|Electricity|Gas:
+   description: Upward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Maximum Ramping|Upwards|Electricity|Nuclear:
+   description: Upward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Maximum Ramping|Upwards|Electricity|Oil:
+   description: Upward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Maximum Ramping|Downwards|Electricity|Coal:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Maximum Ramping|Downwards|Electricity|Gas:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Maximum Ramping|Downwards|Electricity|Nuclear:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+Maximum Ramping|Downwards|Electricity|Oil:
+   description: Downward ramp limit of the power generation unit who belongs to the specified power
+                plant category.
+   unit: GW/h
+
+# description of simple hydrosystems - reservoir hydro
+# this describes systems composed of one reservoir and one plant (with potential pumping capacity) that are used for seasonal storage
+Output Energy of the Second Block|Electricity|Hydro:
```

Comment:
This sounds strange to me - the "second block" is not intuitive...

Reply:
I removed this variable because it is an internal variable that is part of the production variable of a power plant.